package.json

```
@@ -2,33 +2,43 @@
   "name": "@6river/prettier-config",
   "version": "0.0.0-development",
   "description": "6 River Systems Prettier configuration.",
-  "main": "index.json",
+  "homepage": "https://github.com/6RiverSystems/prettier-config#readme",
+  "bugs": {
+    "url": "https://github.com/6RiverSystems/prettier-config/issues"
+  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/6RiverSystems/prettier-config.git"
   },
+  "license": "MIT",
   "contributors": [
     "Adam Vigneaux <avigneaux@6river.com>"
   ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/6RiverSystems/prettier-config/issues"
+  "main": "index.json",
+  "scripts": {
+    "lint": "prettier --check '**/*.{js,ts,md,json,yml,yaml,css,scss,less,graphql,mdx,jsx,tsx}'",
+    "lint:fix": "prettier --write '**/*.{js,ts,md,json,yml,yaml,css,scss,less,graphql,mdx,jsx,tsx}'"
```

nit: no longer necessary to specify a glob like this
```suggestion
    "lint": "prettier --check .",
    "lint:fix": "prettier --write ."
```
[docs](https://prettier.io/docs/en/cli.html)
> This command formats all files supported by Prettier in the current directory and its subdirectories.

---

I don't feel strongly about it, but I'd also point out that [Prettier is not a linter](https://prettier.io/docs/en/comparison.html), so it could be confusing to run it with the `lint` script. I typically do `format:check` and `format:write`, for what it's worth.

---

`-- write .` -- will :eyes: ... not sure if there was a reason we did it with the filename wildcard before

lint vs format: I kinda agree with you, but this would be a yak shave across CI flows/etc., and from the human workflow point of view, format checking and linting are both static analysis, and I think that's what most people really think of from linting. At this point, I see negative value in changing people's workflows here.

plugin as dep: haven't tested that yet, but I'm fairly confident with how pnpm works that it will fail.